### PR TITLE
Sync reset progress marker release study - disable for investigations

### PR DESCRIPTION
**Pull request by AlexeyBarabash**
Apply these values for `ResetProgressMarkerOnCommitFailures` feature 
0% enabled, 100% disabled

for investigating sync server issues

CC @jsecretan , @hspencer77 , @yrliou

### CODE REVIEW

File: seed/seed.json
Comment (moritzhaller) [inline seed/seed.json:544]: Note: since this is enabled by default https://github.com/brave/brave-core/blob/06042dcf838745820a5448c4b66bdd017d14c54e/components/sync/engine/brave_model_type_worker.cc#L21 this PR won't disable on iOS (as we haven't rolled out variations to iOS)

**Review by moritzhaller — APPROVED** [review]
++

File: seed/seed.json
Comment (AlexeyBarabash) [reply seed/seed.json:544]: thanks, I am not expecting lots of affected users on iOS

**Review by kjozwiak — APPROVED** [review]
Approved & Merged after deliberating with @brave/uplift-approvers.

**Pull request merged**
